esmvalcore/local.py

```diff
@@ -414,7 +408,8 @@ def _get_rootpath(project):
                     key, ', '.join(str(p) for p in nonexistent))
                 _ROOTPATH_WARNED.add((key, nonexistent))
             return rootpath[key]
-    raise KeyError('default rootpath must be specified in config-user file')
+    raise KeyError(f'The "{project}" option is missing from the "rootpath" '
+                   'section in the config-user.yml file.')
```

Comment:
The pre-commit hooks made all the changes in this PR, except for this one, which is the one I made :)